frontend/src/api/TeamProject.api.ts

```diff
@@ -0,0 +1,44 @@
+import api from './api.service'
```

Comment:
Remove that file, in my task i already added same file and completely refractored, so its not necessary here

Reply:
:+1: